my_first_web_app/README3A.md

```diff
@@ -0,0 +1,41 @@
+# General info
+
+This project is a simple backend API for schedule website.
+
+
+#### Table of contents
+
+* [Requirements](#requirements)
+* [Instalation](#instalation)
+* [Running the project](#running)
+
+	
+
+
+## Requirements
+
+For development, you will need:
+* Node.js and a node global package
+* Frameworks/Librarys:
+    - "express" version: "^4.17.3",
+    - "joi" version: "^17.6.0",
+    - "js-sha256" version: "^0.9.0"
+
```

Comment:
I saw this already, 
1) What's node global package?
2) Do not list here package versions; they're fine in package.json